src/pages/nonprofit/issues/index.tsx

```diff
@@ -1,9 +1,184 @@
-import { Flex } from "@chakra-ui/react";
+import {
+  Button,
+  Flex,
+  Heading,
+  Text,
+  VStack,
+  Tabs,
+  Tab,
+  TabList,
+  TabPanels,
+  TabPanel,
+} from "@chakra-ui/react";
+import Link from "next/link";
+import { useEffect, useState } from "react";
+import { nonprofitGetIssues, nonprofitGetProjects } from "src/actions/Project";
+import IssuesTable from "src/components/nonprofit/issues/IssuesTable";
+import MaintenanceProjectCard from "src/components/nonprofit/issues/MaintenanceProjectCard";
+import { dateDiffInDays, dateToMMDDYYYY } from "src/utils/dates";
+import { showError } from "src/utils/notifications";
+import {
+  Issue,
+  IssueStatus,
+  Chapter,
+  Project,
+  ProjectStage,
+} from "src/utils/types";
+import urls from "src/utils/urls";
 
 function NonprofitIssuesPage() {
+  const [projects, setProjects] = useState<Project[]>();
+  const [currProject, setCurrProject] = useState<Project>();
+  const [issues, setIssues] = useState<Issue[]>();
+
+  useEffect(() => {
+    async function fetchProjects() {
+      let projects: Project[] = await nonprofitGetProjects({
+        status: ProjectStage.MAINTENANCE,
+      });
+
+      projects = projects.map((project) => {
+        // maintenance start will never be undefined if stage is maintenance
+        const endDate = new Date(project.maintenanceStart!);
+        endDate.setDate(
+          endDate.getDate() + (project.chapter as Chapter).maintenancePeriod
+        );
+        return {
+          ...project,
+          maintenanceEnd: endDate,
+          remainingDays: dateDiffInDays(new Date(), endDate),
+        };
+      });
+      if (projects.length !== 0) {
+        setCurrProject(projects[0]);
+      }
+      setProjects(projects);
+    }
+
+    fetchProjects().catch((error: Error) => showError(error.message));
+  }, []);
+
+  useEffect(() => {
+    async function fetchIssues() {
+      if (currProject) {
+        const issues: Issue[] = await nonprofitGetIssues(
```

Comment:
Add status filter to this data flow so that open and closed issues are fetched separately.  I did not make this clear in the issue so I'll take care of this.

Reply:
not sure how to implement this - letting you handle it!